.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "*.{js,jsx,ts,tsx,json,css,scss,md,yml}": "prettier --write"
```

Comment:
```suggestion
  "*.{js,jsx,ts,tsx,json,css,scss,md,yml}": "yarn prettier --write"
```

Reply:
I add this file according to Lintstage's document and it works properly.

Reply:
In this case prettier run from global that may have different version of our project. when we add yarn we urge that use our prettier.